assets/javascripts/discourse/initializers/extend-for-assigns.js

```diff
@@ -860,6 +860,8 @@ export default {
 
       api.addUserSearchOption("assignableGroups");
 
+      api.addSaveableUserOptionField("notification_level_when_assigned");
```

Comment:
We need to "globally" allow this user option to be "saveable" when save a user preference's page.